image/db/rhel/scripts/docker-entrypoint.sh

```diff
@@ -33,19 +33,20 @@ _is_sourced() {
 		&& [ "${FUNCNAME[1]}" = 'source' ]
 }
 
-# used to create initial posgres directories and if run as root, ensure ownership to the "postgres" user
+# used to create initial postgres directories and if run as root, ensure ownership to the "postgres" user
 docker_create_db_directories() {
 	local user; user="$(id -u)"
 
 	mkdir -p "$PGDATA"
-	chmod 700 "$PGDATA"
+	# ignore failure since there are cases where we can't chmod (and PostgreSQL might fail later anyhow - it's picky about permissions of this directory)
+	chmod 700 "$PGDATA" || :
```

Comment:
Did you see real error for this?

Reply:
No, this is just directly lifted from https://github.com/docker-library/postgres/blob/master/12/bullseye/docker-entrypoint.sh

We haven't updated this file in years, so I just copy pasted the latest version here and added comments in sections marked with `STACKROX MODIFIED`